### PR TITLE
HasAttachmentTrait were added to reduce code duplication

### PR DESCRIPTION
**Pull request by hlogeon**
I found code copy\paste. O know that it's not critical but I believe it's better to avoid such things.

### CODE REVIEW

File: HasAttachmentTrait.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<?php
+/**
```
Comment (PEMapModder) [inline HasAttachmentTrait.php]: Remove this?

Comment (hlogeon) [inline HasAttachmentTrait.php]: Why? I see that all other files has opening tags on the first line. Or do you mean to make comment block to match all other files?

Comment (PEMapModder) [inline HasAttachmentTrait.php]: Yes, like other files.

Comment (hlogeon) [inline HasAttachmentTrait.php]: Fixed

**Comment by hlogeon** [comment]
Hi, I just want to notify you about the pull-request

**Pull request closed without merging**